server/index.js

```diff
@@ -26,7 +26,12 @@ io.on('connection', function(socket) {
 		io.emit('movement', event);
 	});
 
-	socket.on('disconnect', function() {
+
+	socket.on('new snake', snakeData => {
+		socket.broadcast.emit('snakeData', snakeData);
```

Comment:
> io.emit?

Answered orally. `socket.broadcast` broadcasts to everyone but the socket's client (which doesn't need to know what it sends)